source/ShipInfoDisplay.h

```diff
@@ -41,14 +41,14 @@ class ShipInfoDisplay : public ItemInfoDisplay {
 	// int MaximumHeight() const;
 	// int DescriptionHeight() const;
 	// int AttributesHeight() const;
+	int GetAttributesHeight(bool sale) const;
 	int OutfitsHeight() const;
-	int SaleHeight() const;
 	
 	// Provided by ItemInfoDisplay:
 	// void DrawDescription(const Point &topLeft) const;
 	virtual void DrawAttributes(const Point &topLeft) const override;
+	virtual void DrawAttributes(const Point &topLeft, const bool sale) const;
```

Comment:
Instead of having two different DrawAttributes functions, you should have only one function with a default sale value of false. That way you avoid the need of the first function to just call the second function.